spigot/src/main/java/org/geysermc/floodgate/util/WhitelistUtils.java

```diff
@@ -25,10 +25,17 @@
 
 package org.geysermc.floodgate.util;
 
+import static org.bukkit.Bukkit.getServer;
+
 import com.mojang.authlib.GameProfile;
 import java.util.UUID;
 import org.bukkit.Bukkit;
 import org.bukkit.OfflinePlayer;
+import org.geysermc.floodgate.FloodgatePlatform;
+import org.geysermc.floodgate.SpigotPlugin;
+import org.geysermc.floodgate.api.FloodgateApi;
```

Comment:
Is this (and the other new imports) needed?
Doesn't look like they're used

Reply:
You're right. That was accidentally left over from when I was trying to figure out a solution. I've removed them.